renderer/src/test/welcome_onboardingCompleted.test copy.tsx

```diff
@@ -0,0 +1,47 @@
+import { beforeEach, describe, expect, test, vi } from 'vitest'
```

Comment:
this test file has an unexpected name, is the ` copy` intentional?

Reply:
nop